src/transformers/models/mobilevitv2/modeling_mobilevitv2.py

```diff
@@ -58,7 +58,7 @@
 
 
 MOBILEVITV2_PRETRAINED_MODEL_ARCHIVE_LIST = [
-    "apple/mobilevitv2-1.0-imagenet1k-256"
+    "apple/mobilevitv2-1.0-imagenet1k-256" "apple/mobilevitv2-1.0-voc-deeplabv3"
```

Comment:
These shouldn't be changed

Reply:
I have removed it in 1669db11280ecadd33b7535f92241b4877b15393